pyomo/core/base/reference.py

```diff
@@ -348,62 +387,103 @@ def _get_iter(self, _slice, key):
 
 
 def _identify_wildcard_sets(iter_stack, index):
-    # if we have already decided that there isn't a comon index for the
+    # if we have already decided that there isn't a common index for the
     # slices, there is nothing more we can do.  Bail.
     if index is None:
         return index
 
     # Walk the iter_stack that led to the current item and try to
     # identify the component wildcard sets
+    #
+    # `tmp` will be a list of dicts mirroring the iter_stack. Each
```

Comment:
We should probably rename `tmp` to be something more meaningful - as it holds the ultimate answer!

Reply:
Have renamed to `wildcard_stack`.